shell/src/Luna/Shell/Interpret.hs

```diff
@@ -8,6 +8,7 @@ import qualified Data.Graph.Data.Graph.Class         as Graph
 import qualified Data.Map                            as Map
 import qualified Luna.IR                             as IR
 import qualified Luna.Package                        as Package
+import qualified Luna.Package.Env                    as PackageEnv
```

Comment:
You could just call this `Package` as there aren't any name clashes.

Reply:
Counterpoint: I want to be able to unambiguously go to the file defining the function being used.